bin/now-deploy.js

```diff
@@ -417,6 +448,13 @@ async function sync({ token, config: { currentTeam, user } }) {
 
   // Merge `now.env` from package.json with `-e` arguments.
   const pkgEnv = nowConfig && nowConfig.env
+
+  if (Array.isArray(nowConfig.env)) {
```

Comment:
This breaks deployments that don't have a `now` on `package.json` or a `now.json`:
![image](https://user-images.githubusercontent.com/4721750/26944564-f0a6c288-4c3d-11e7-9435-9bfa7e727951.png)